lib/geocoder/lookups/ipinfo_io.rb

```diff
@@ -0,0 +1,41 @@
+require 'geocoder/lookups/base'
+require 'geocoder/results/ipinfo_io'
+
+module Geocoder::Lookup
+  class IpinfoIo < Base
+
+    def name
+      "Ipinfo.io"
+    end
+
+    def query_url(query)
+      "#{protocol}://ipinfo.io/#{query.sanitized_text}/geo"
+    end
+
+    # currently doesn't support HTTPS
```

Comment:
I think we should not restrict this on our side since paid plans actually allow https.
See http://ipinfo.io/pricing

We can make sure the tests skip https check if they fail due to this reason.